### PR TITLE
Feat/tokens extension

### PR DESCRIPTION
**Pull request by tomasklim**
## Description

- using types from blockbook api type file
- change token `address` to `contract` in suite
- `ERC1155` and `ERC721` are not filtered in blockchain-link
- use sort token logic also for send form

## Related Issue

Successor of #7855 handling tokens 

Part II of #8076

### CODE REVIEW

File: packages/blockchain-link-types/src/blockfrost.ts [hunk 1]
```diff
@@ -216,6 +216,15 @@ export interface AssetBalance {
     fingerprint?: string; // defined for all assets except lovelace
 }
 
+export type BlockfrostToken = {
```
Comment (mroz22) [inline packages/blockchain-link-types/src/blockfrost.ts:219]: I would expect seeing implementation in this commit

File: packages/blockchain-link-types/src/common.ts [hunk 2]
```diff
@@ -176,7 +167,7 @@ export interface Utxo {
 
 export interface TokenInfo {
     type: string; // token type: ERC20...
-    address: string; // token address
+    contract: string; // token address
```
Comment (mroz22) [inline packages/blockchain-link-types/src/common.ts:170]: migrations might be needed?

File: packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```
Comment (mroz22) [inline packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx]: oooo

Comment (tomasklim) [reply packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/index.tsx]: 🤯

File: packages/blockchain-link-types/src/common.ts (hunk 2 above)
Comment (tomasklim) [reply packages/blockchain-link-types/src/common.ts:170]: done

**Review by mroz22 — APPROVED** [review]

**Pull request merged**